From 786aa237b65ccc9b6de351fc4963bc418e2b4612 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Wed, 3 May 2017 17:31:48 +0000
Subject: [PATCH 171/214] www: Minor corrections

 * Samsung Galaxy S III (d2) was never publicly considered as removed
 * Add the device model names to the text for reading consistency
 * Fix miscellaneous markdown across some old posts

Change-Id: I80f1ca1d674d7bba283a5390c9378ad5a3900389
---
 _posts/2017-02-20-Last-Week-in-LineageOS-2.md    | 2 ++
 _posts/2017-04-01-Announcing-our-Mascot.md       | 1 +
 _posts/2017-04-04-Last-Week-in-LineageOS-6.md    | 1 +
 _posts/2017-04-18-Last-2-Weeks-in-LineageOS-7.md | 2 +-
 _posts/2017-05-02-Changelog-8.md                 | 6 +-----
 5 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/_posts/2017-02-20-Last-Week-in-LineageOS-2.md b/_posts/2017-02-20-Last-Week-in-LineageOS-2.md
index 379103c..3e48160 100644
--- a/_posts/2017-02-20-Last-Week-in-LineageOS-2.md
+++ b/_posts/2017-02-20-Last-Week-in-LineageOS-2.md
@@ -19,8 +19,10 @@ author: harryyoud
 When we started developing for Android 7.0 (not 7.1), we based our source code on CAF 7.0 (CAF we depend upon for almost all our Qualcomm based devices).  
 When Android 7.1 was released to AOSP, we merged this on top, but kept CAF 7.0 for HALs. Now CAF have updated their repositories to Android 7.1, we have rewritten history and rebased all merged commits on top of the CAF 7.1 base.
 This means:
+
 * much less time spent resolving merge conflicts
 * AOSP's February security patches (AKA r21) can be merged much more easily into LineageOS 14.1 (which is why we're later than usual for this month's security patches).  
+
 Before we did the huge rebase (happening over the past week), we created branches called `cm-14.1_prerebase` to make sure we had a known good copy to go back to, just in case the rebase did not go successfully.  
 We'll be working on incorporating the Feb security release into 13.0 builds in the near future.
 
diff --git a/_posts/2017-04-01-Announcing-our-Mascot.md b/_posts/2017-04-01-Announcing-our-Mascot.md
index 77cc115..743febb 100644
--- a/_posts/2017-04-01-Announcing-our-Mascot.md
+++ b/_posts/2017-04-01-Announcing-our-Mascot.md
@@ -16,6 +16,7 @@ Please take note of the date this was posted. This post was meant to be satire.
 
 ## New Mascot
 We've seen an inordinate amount of people asking two questions: 
+
  > "_What is the LineageOS Mascot?_"
 
  > "_Where is VoLTE?_"
diff --git a/_posts/2017-04-04-Last-Week-in-LineageOS-6.md b/_posts/2017-04-04-Last-Week-in-LineageOS-6.md
index 17973dc..efaf990 100644
--- a/_posts/2017-04-04-Last-Week-in-LineageOS-6.md
+++ b/_posts/2017-04-04-Last-Week-in-LineageOS-6.md
@@ -12,6 +12,7 @@ author: harryyoud
 We're glad so many of you have enjoyed (and fallen for) our April Fool's antics. 
 
 To be absolutely clear:
+
 * VoLTE is the name of our (temporary) mascot (who is retiring on Friday). If you haven't seen it yet, check out our [blog post](http://lineageos.org/Announcing-our-Mascot/).
 * br0zip is a harmless just-for-fun app shipping on LineageOS 14.1 builds from 3rd April to 7th April (_Disclaimer: It does not actually produce functional ROMs_)
 * VoLTE, our mascot, is also in the bootanimation on LineageOS 14.1 builds from 3rd April to 7th April
diff --git a/_posts/2017-04-18-Last-2-Weeks-in-LineageOS-7.md b/_posts/2017-04-18-Last-2-Weeks-in-LineageOS-7.md
index 1c22128..d3a06dd 100644
--- a/_posts/2017-04-18-Last-2-Weeks-in-LineageOS-7.md
+++ b/_posts/2017-04-18-Last-2-Weeks-in-LineageOS-7.md
@@ -39,7 +39,7 @@ Added 14.1 devices
 Changes to 14.1 devices
 
 * Samsung Galaxy Note 2 LTE Korean variants (t0ltektt,t0lteskt) are now unified under t0ltekor - _maintainers: filiprrs, PoisonNinja_
-* klteusc has been merged into klte. To continue receiving updates, flash the latest klte build, as this is the last week of klteusc builds
+* Samsung Galaxy S5 US Celullar (klteusc) has been merged into Samsung Galaxy S5 (klte). To continue receiving updates, flash the latest klte build, as this is the last week of klteusc builds.
 
 Removed 14.1 devices
 
diff --git a/_posts/2017-05-02-Changelog-8.md b/_posts/2017-05-02-Changelog-8.md
index 6f6106c..c828de6 100644
--- a/_posts/2017-05-02-Changelog-8.md
+++ b/_posts/2017-05-02-Changelog-8.md
@@ -35,10 +35,6 @@ Added 14.1 devices
 * [Samsung Galaxy S4 Mini (International Dual SIM) - serranodsdd](https://wiki.lineageos.org/devices/serranodsdd) - _maintainer: arco_
 * [HTC One A9 (International GSM) - hiaeuhl](https://wiki.lineageos.org/devices/hiaeuhl) - _maintainer: intervigil_
 * [HTC One A9 (US GSM) - hiaeul](https://wiki.lineageos.org/devices/hiaeul) - _maintainer: intervigil_
-* [Samsung Galaxy S III (AT&T) - d2att](https://wiki.lineageos.org/devices/d2att) - _maintainer: javelinanddart_
-* [Samsung Galaxy S III (Sprint) - d2spr](https://wiki.lineageos.org/devices/d2spr) - _maintainer: javelinanddart_
-* [Samsung Galaxy S III (T-Mobile) - d2tmo](https://wiki.lineageos.org/devices/d2tmo) - _maintainer: javelinanddart_
-* [Samsung Galaxy S III (Verizon) - d2vzw](https://wiki.lineageos.org/devices/d2vzw) - _maintainers: javelinanddart, invisiblek_
 * [Samsung Galaxy Tab S 10.5 Wi-Fi - chagallwifi](https://wiki.lineageos.org/devices/chagallwifi) - _maintainers: deadman96385, DarkExistence_
 * [Samsung Galaxy Note 3 (International 3G) - ha3g](https://wiki.lineageos.org/devices/ha3g) - _maintainers: tincho5588, deadman96385_
 * [LG V20 (AT&T) - h910](https://wiki.lineageos.org/devices/h910) - _maintainer: albinoman887_
@@ -54,4 +50,4 @@ Changes to 14.1 devices
 
 Removed 14.1 devices
 
-* [Sony Xperia M - nicki](https://wiki.lineageos.org/devices/nicki) - No longer maintained
+* [Sony Xperia M - nicki](https://wiki.lineageos.org/devices/nicki) - no longer maintained
-- 
2.17.1

